ballerina/tests/fromXml_test.bal

```diff
@@ -3144,3 +3144,47 @@ function testUnsupportedTypeNegative() {
     |}|error err3 = parseAsType(xmlVal2);
     test:assertEquals((<error>err3).message(), "unsupported input type");
 }
+
+@Namespace {
+    uri: "http://example.com/book"
+}
+type OpenBook record {
```

Comment:
Shall we add a test for open but nested records?

Reply:
Added [Add test with open record with nested record](https://github.com/ballerina-platform/module-ballerina-data.xmldata/pull/16/commits/326535a04ef14987a7d01c3b9984cbda49082646)